src/vuestic-theme/vuestic-components/vuestic-datatable/datatable-components/FilterBar.vue

```diff
@@ -18,20 +18,26 @@ export default {
     label: {
       type: String,
     },
+    value: ''
   },
   data () {
     return {
-      filterText: '',
       typingTimeout: null,
       typingDelay: 250,
     }
   },
   methods: {
-    doFilter () {
```

Comment:
You removed delay feature. Please check it stays sane in places where it's used.